src/components/dashboard/sidebar/LearnerCreditSummaryCard.jsx

```diff
@@ -46,100 +45,138 @@ const badgeStatusMessages = defineMessages({
+            />
+          )}
+        </p>
+
+        {(expirationDate && !disableExpiryMessagingForLearnerCredit) && (
+          <p className="mb-0" data-testid="learner-credit-summary-end-date-text">
+            {isBudgetExpired ? (
               <FormattedMessage
-                id="enterprise.dashboard.sidebar.learner.credit.card.title"
-                defaultMessage="Learner Credit"
-                description="Title for the learner credit summary card on the enterprise dashboard sidebar."
+                id="enterprise.dashboard.sidebar.learner.credit.card.subsidy.expired.date"
+                defaultMessage="Expired on {subsidyExpiryDate}"
+                description="Subsidy expired date for the learner credit summary card on the enterprise dashboard sidebar."
+                values={{
+                  subsidyExpiryDate: (
+                    <b>
+                      <FormattedDate
+                        value={expirationDate}
+                        year="numeric"
+                        month="short"
+                        day="numeric"
+                      />
+                    </b>
+                  ),
+                }}
               />
-            </h3>
-            {cardBadge}
-          </div>
-        )
-      }
-      cardClassNames={className}
-    >
-      <p data-testid="learner-credit-summary-text">
-        {assignmentOnlyLearner ? (
-          <FormattedMessage
-            id="enterprise.dashboard.sidebar.learner.credit.card.assignment.only.description"
-            defaultMessage="Your organization will assign courses to learners. Please contact your administrator if you are interested in taking a course."
-            description="Description for the learner credit summary card on the enterprise dashboard sidebar when learner has assignment."
-          />
-        ) : (
-          <FormattedMessage
-            id="enterprise.dashboard.sidebar.learner.credit.card.description"
-            defaultMessage="Apply your organization's learner credit balance to enroll into courses with no out of pocket cost."
-            description="Description for the learner credit summary card on the enterprise dashboard sidebar when learner has no assignment."
-          />
+            ) : (
+              <FormattedMessage
+                id="enterprise.dashboard.sidebar.learner.credit.card.subsidy.expiration.date"
+                defaultMessage="Available until {subsidyExpiryDate}"
+                description="Subsidy expiration date for the learner credit summary card on the enterprise dashboard sidebar."
+                values={{
+                  subsidyExpiryDate: (
+                    <b>
+                      <FormattedDate
+                        value={expirationDate}
+                        year="numeric"
+                        month="short"
+                        day="numeric"
+                      />
```

Comment:
[nit/suggestion] Would it make sense to use `i18nFormatTimestamp` here so this date formatting is isolated to one place versus replicated here?

```jsx
<FormattedMessage
  id="enterprise.dashboard.sidebar.learner.credit.card.subsidy.expiration.date"
  defaultMessage="Available until {subsidyExpiryDate}"
  values={{
    subsidyExpiryDate: (
      <b>{i18nFormatTimestamp({ intl, timestamp: expirationDate })}</b>
    ),
  }}
/>
```